internal/services/postgres/postgresql_flexible_server_configuration_resource.go

```diff
@@ -136,10 +136,7 @@ func resourceFlexibleServerConfigurationRead(d *pluginsdk.ResourceData, meta int
 
 	d.Set("name", id.ConfigurationName)
 	d.Set("server_id", configurations.NewFlexibleServerID(subscriptionId, id.ResourceGroupName, id.FlexibleServerName).ID())
-
-	if resp.Model != nil && resp.Model.Properties != nil {
-		d.Set("value", resp.Model.Properties.Value)
-	}
+	d.Set("value", d.Get("value").(string))
```

Comment:
This means we won't be tracking this value if it changes in the remote API, so this change isn't correct

Unfortunately the ordering of some fields is important, so we can't fix this via a DiffSuppressFunc, the fix for this would be to update the config to specify the correct value here?

Reply:
OK. Thanks for suggestion. I will close this PR.